README.md

```diff
@@ -195,6 +195,14 @@ metrics = require('metrics')
   * `num` Value to put in histogram (number).
   * `label_pairs` Table containing label names as keys, label values as values (table). New value is observed by all internal counters with these labels specified.
 
+#### `histogram_obj:observe_latency(label_pairs, fn, ...)`
```

Comment:
Don't forget to add corresponding changelog note

Reply:
Done